src/shared/ad-options.interface.ts

```diff
@@ -35,4 +35,22 @@ export interface AdOptions {
    * @since 1.2.0
    */
   npa?: boolean;
+
+  /**
+   * For rewarded Ads SSV only.
+   * Set this value to send a user identifier to your rewarded Ads SSV call.
+   * 
+   * @default undefined
+   * @since 3.0.1
+   */
+  userId?: string;
+
+  /**
+   * For rewarded ads SSV only.
+   * Set this value to send custom data to your rewarded ads SSV call.
+   * 
+   * @default undefined
+   * @since 3.0.1
+   */
+  customData?: string;

```

Comment:
Since this interface is extended by all the ad types, I think will be better to add it to a new file on `src/reward/reward-ad-options.interface.ts`

```
export interface RewardAdOptions extends AdOptions {
    /**
     * For rewarded Ads SSV only.
     * Set this value to send a user identifier to your rewarded Ads SSV call.
     * 
     * @default undefined
     * @since 3.0.1
     */
    userId?: string;
  
    /**
     * For rewarded ads SSV only.
     * Set this value to send custom data to your rewarded ads SSV call.
     * 
     * @default undefined
     * @since 3.0.1
     */
    customData?: string;
}
```
and please also update `reward-definitions.interface.ts` to use the new interface
https://github.com/capacitor-community/admob/blob/47280fa329491715ecb0b5dfbe80987f5657e2da/src/reward/reward-definitions.interface.ts#L19-L21